.github/workflows/iOSBuild.yml

```diff
@@ -0,0 +1,40 @@
+name: iOS Build
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+
+concurrency:
+  group: ios-build-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build-ios:
+    runs-on: macos-13
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
+
+      - name: Select Xcode
+        run: sudo xcode-select -s '/Applications/Xcode_15.0.app'
+
+      - uses: ./.github/actions/setup-java
+
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@250fcd6a742febb1123a77a841497ccaa8b9e939 # v1.152.0
+        with:
+          bundler-cache: true
+
```

Comment:
could you put arch to `local.properties` to reduce build time?



reference is here!

https://github.com/DroidKaigi/conference-app-2023/blob/main/build-logic/src/main/kotlin/io/github/droidkaigi/confsched2023/primitive/SourceSet.kt#L6

Reply:
done! https://github.com/DroidKaigi/conference-app-2023/pull/221/commits/fd332b563b529340ce74d765702694effb513209